docs/UserGuide.md

```diff
@@ -97,46 +97,50 @@ Expressions without `...` at the end takes the last parameter as the argument wh
 ### Adding a tag with filepath : `tag`
 
 Tags a file with a unique tag name.<br>
-You can add a nickname to a file. Optionally, you can add some labels to the tag for categorisation.
-The `FILE_PATH` field can accept both absolute and relative file path from your current directory in HelloFile.
+You can add a nickname to a file. This make it easier to manage only some specific files that you want such as frequently used files, etc.
```

Comment:
Maybe should be "makes"? and personally I think "easier to manage some files" is a bit abstract.

Reply:
Yeah, there is a grammar mistake. However, I want to put a reason on why you would want to use tag command. Is "makes you able to manage your files easily" is better? Or do you have any suggestion?

Reply:
How about "You can add a nickname to a file which can be used to access the file easily"? But I think "makes you able to manage your files easily" is also fine.

Reply:
Okay. Changed!